test/integration/backward_compatible/HazelcastClientTest.js

```diff
@@ -116,44 +116,31 @@ configParams.forEach(function (cfg) {
             expect(info.labels).to.deep.equal(new Set());
         });
 
-        it('getDistributedObjects returns all dist objects', function (done) {
+        it('getDistributedObjects returns all dist objects', function () {
             managed.getObject(client.getMap.bind(client, 'map'));
             managed.getObject(client.getSet.bind(client, 'set'));
-            setTimeout(function () {
-                client.getDistributedObjects().then(function (distObjects) {
-                    try {
-                        const names = distObjects.map((o) => {
-                            return o.getName();
-                        });
-                        expect(names).to.have.members(['map', 'set']);
-                        done();
-                    } catch (e) {
-                        done(e);
-                    }
+            return TestUtil.assertTrueEventually(async function () {
```

Comment:
Just to be consistent with the rest of the tests, can we use arrow functions there?